src/Files.App/App.xaml.cs

```diff
@@ -18,6 +18,38 @@ namespace Files.App
 	/// </summary>
 	public partial class App : Application
 	{
+		internal readonly static string AppInformationKey = @$"Software\Files Community\{Package.Current.Id.Name}\v1\AppInformation";
+
+		public static bool IsAppUpdated { get; }
+		public static bool IsFirstRun { get; }
+		public static long TotalLaunchCount { get; }
+
+		static App()
+		{
+			using var infoKey = Registry.CurrentUser.CreateSubKey(AppInformationKey);
```

Comment:
Is this the virtual registry? Also, it would be preferable to create a helper class.

Reply:
Yeah it's virtual registry.

Reply:
Is there any reason not to use our existing settings service?

Reply:
`Information.*` APIs are no longer available in WCT 8.x.

Reply:
> so if multiple Files instances launch simultaneously, there won't be any read-write conflicts.

This is good enough reason to use the registry, however, we should put it in a separate helper for code quality.